src/ui/controls/edit.cpp

```diff
@@ -84,7 +84,17 @@ bool IsSep(int character)
     return !IsWord(character);
 }
 
+bool IsBreaker(char c)
+{
+    return ( c == '.'  || c == '{' || c == '}' ||
+             c == ';' || c == ':' || c == '[' || c == ']' ||
+             c == '(' || c == ')' || c == '=' );
```

Comment:
l think = shouldn't be a breaker, +,-,/ etc are also no breakers and using = without spaces is bad code style, isn't it?